src/setStorage.ts

```
@@ -1,24 +1,35 @@
+    uni.addInterceptor("setStorage", storageInvokeInterceptor);
   },
 };
+
+/**
+ * 直接应用快手小程序setStorage代理修复拦截器
+ * 可以作为 Vue 插件使用: Vue.use(KuaiShouSetStorageProxyFixInterceptor)
+ * 也可以直接调用: applyKuaiShouSetStorageProxyFixInterceptor()
+ */
+export function applyKuaiShouSetStorageProxyFixInterceptor(): void {
+  KuaiShouSetStorageProxyFixInterceptor.install();
+}
```

ESLint

[error] Newline required at end of file but not found.

(style/eol-last)